.storybook/config.js

```diff
@@ -25,12 +27,17 @@ function loadStories() {
 const context = require.context('../components', true, /\.twig$/)
 context.keys().forEach(key => {
   var template = context(key);
+  const rootPath = path.join(__dirname);
   Twig.twig({
+    base: rootPath, // Possibly can set a base directory for everything to work from?
```

Comment:
This was something added in older versions of Twig.js that seemed to allow it to set a base directory for calling all the files. I'm not sure if it is supported and/or needed here.